### ISSUE TITLE
Ghost footer needs tweaks

### ISSUE THREAD

**Issue by JosephMcArthur** [oaworks/casper-rework#1]
See here: https://blog.oa.works/open-access-button-is-now-oa-works/

Need to:

- [x] add cc licensing
- [x] remove `powered by ghost`
- [x] filter out posts with `talk` in them from the feed

**Comment by JosephMcArthur** [comment]
> > filter out posts with `talk` in them from the feed
> 
> What are those?

When I do talks I share my slides. I used to them make a page on the site where those slides could be posted with a nice URL & grouped with relevant calls to action and resources. Now we're in the process of retiring the current OAB site I thought I could move those to the blog, as that site feels like a better fit for content like that. However, I don't want to fill up our homepage with dozens of talks - so I made a tag for them (`talk`) where they could be filtered off the blog homepage (and the feed on oa.works. The only place that I didn't manage to do it yet was the feed in the footer, which [seems straightforward](https://forum.ghost.org/t/how-to-exclude-specific-posts-from-the-home-page/11010) but not without touching the code.
